### DISCUSSION TITLE
/ct public and /at public not working

### DISCUSSION THREAD

**Discussion by sophie462** [GriefPrevention/GriefPrevention#890] [Q&A] [answered]
Even when I use /ct public or /at public, non-opped players get the message: You are not permitted to interact with " "
I tried to take out plugins to see if one of them was the problem, and when I removed EssentialsX, /ct and /at worked again. I know EssentialsX doesn't recognize @'s, so does it not recognize the use of 'public' either? I have CommandHook, but /at @a or /ct @a are not commands. Is there a way around this? 

Update: I have tried /containertrust playername and /accesstrust playername and the player still cannot access objects.

**Comment by RoboMWM** [comment]
no idea what you're talking about. Sounds like plugins are interfering/registering these commands. Not sure what CommandHook is.

**Comment by sophie462** [comment]
CommandHook is a plugin that registers the @'s for Essentials. Do you have any way to fix this problem? I asked Essentials support, and they that since Essentials does not have any claiming commands, it is most likely a GriefPrevention problem. Is there any way I can give all players containertrust and accesstrust?

**Comment by sophie462** [comment]
![image](https://user-images.githubusercontent.com/68306607/87862217-7d729400-c902-11ea-9ab6-d3522973e6ec.png)
This message shows even after I have done all the permission commands.

**Comment by Jikoo** [comment]
Those messages are [from EssentialsAntiBuild](https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/messages.properties#L19), not GP. You probably need to configure permissions for it, but for that you should seek help from the EssentialsX team or (assuming you know what nodes you want to apply to users and are struggling with that) your permission provider.

**Comment by sophie462** [comment]
I found a solution. Thank you.

**Comment by RoboMWM** [comment]
Would you mind sharing the solution for any others who may come across this issue?

**Comment by sophie462** [comment, marked as answer]
Turns out, it was a problem in the Essentials jar files. I am pretty sure the .jar files Anti-Build and Protection are the ones that were overriding the permissions commands. 
To let players access spawn, you just need to remove those .jar files.
